scopes/component/component/component.ui.runtime.tsx

```diff
@@ -79,11 +79,16 @@ export class ComponentUI {
    */
   private activeComponent?: ComponentModel;
 
+  formatToInstallableVersion(version: string) {
+    if (semver.valid(version)) return version;
```

Comment:
not sure if it'll work for all cases, e.g. `semver.valid('^1.0.0');` returns null.
maybe you can use `snapToSemver` function from component-package-version

Reply:
updated to reuse the `snapToSemver` function for consistency. But, Component versions cannot be a range, only valid semver or a hash